docs/central-install-digital-ocean.rst

```diff
@@ -320,40 +320,40 @@ Using a Custom Database Server
 ------------------------------
 
 .. warning::
-  Using a custom database server, especially one that is not local to your local network, may result in poor performance. We strongly recommend using the Postgres v9.6 server that is bundled with Central.
+  Using a custom database server, especially one that is not local to your local network, may result in poor performance. We strongly recommend using the Postgres v14 server that is bundled with Central.
 
 Central ships with a PostgreSQL database server. To use your own custom database server:
 
 1. Ensure you have a PostgreSQL database server visible to your Central server network host.
 2. Ensure your database has ``UTF8`` encoding by running the following command on the database.
 
-  .. code-block:: console
+   .. code-block:: postgres
 
-    SHOW SERVER_ENCODING;
+      SHOW SERVER_ENCODING;
 
 3. Ensure ``CITEXT`` and ``pg_trgm`` extensions exist by running the following commands on the database.
 
-  .. code-block:: console
+   .. code-block:: postgres
 
-    CREATE EXTENSION IF NOT EXISTS CITEXT;
-    CREATE EXTENSION IF NOT EXISTS pg_trgm;
+      CREATE EXTENSION IF NOT EXISTS CITEXT;
+      CREATE EXTENSION IF NOT EXISTS pg_trgm;
 
 4. Edit the file ``files/service/config.json.template`` to reflect your database host, table, and authentication details.
 
-  .. code-block:: console
+   .. code-block:: json
 
-    "database": {
-      "host": "my-db-host",
-      "user": "my-db-user",
-      "password": "my-db-password",
-      "database": "my-db-table"
-    },
+      "database": {
```

Comment:
I haven't read through the part where the config changes but I think this and some of the other nearby instructions are going to be different...

Reply:
Addressed.